src/lib/gravity/index.ts

```diff
@@ -6,10 +6,30 @@ class Gravity {
     production: "api.artsy.net",
   }
 
+  async getAccessToken(credentials: Credentials) {
+    const gravityUrl = this.url("oauth2/access_token")
```

Comment:
@joeyAghion this seems like a reasonable way to authenticate, but let me know if there is a method that makes more sense.

Reply:
I'd have to dig... is this where Force posts credentials to obtain an access token today?

Reply:
I _think_ it [is](https://github.com/artsy/artsy-passport/blob/1790a27d572a957c409939a3dc27014092430ba1/lib/passport/callbacks.coffee#L14). And I imagine this cli is not tight to Force, as long as it's a valid access token generated by Gravity?

Reply:
Correct - any valid id/secret will do.